spec/acceptance/nodesets/docker/PuppetserverDockerfile

```diff
@@ -1,6 +1,6 @@
 FROM certs:latest as certs
 
-FROM puppet/puppetserver:6.14.1
+FROM puppet/puppetserver:edge
```

Comment:
I'm not sure how the tests set up 6/7.x branches for the spec testing, but locally testing on edge works for me.